src/Compilers/CSharp/Test/Syntax/Parsing/CollectionLiteralParsingTests.cs

```diff
@@ -0,0 +1,1368 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class CollectionLiteralParsingTests : ParsingTests
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

Consider including a simple test with language version so it's clear parsing does not depend on language version:
```csharp
[Theory]
[InlineData(LanguageVersion.CSharp11)]
[InlineData(LanguageVersion.Preview)]
public void CollectionLiteralExpression(LanguageVersion languageVersion)
{
    UsingExpression("[A, B]", lTestOptions.Regular.WithLanguageVersion(languageVersion));

    N(SyntaxKind.CollectionCreationExpression);
    {
        ...
    }
    EOF();
}
```

Reply:
Done.